devicetypes/smartthings/zwave-metering-switch.src/zwave-metering-switch.groovy

```diff
@@ -284,10 +285,6 @@ def configure() {
 }
 
 def reset() {
-	resetEnergyMeter()
-}
-
-def resetEnergyMeter() {
```

Comment:
Why did You change that ?

Reply:
Sorry, I made the changes based on the master, but I forgot that this master was forked by me, not the latest.